### PR TITLE
fix wrong description

### PR DESCRIPTION
**Pull request by yanggangtony**
Signed-off-by: yanggang <gang.yang@daocloud.io>

#### What type of PR is this?
/kind documentation

#### Special notes for your reviewer:
Read the codes , but confused with the note message..

### CODE REVIEW

File: staging/src/k8s.io/apimachinery/pkg/watch/watch.go [hunk 1]
```diff
@@ -27,7 +27,7 @@ import (
 
 // Interface can be implemented by anything that knows how to watch and report changes.
 type Interface interface {
-	// Stop stops watching. Will close the channel returned by ResultChan(). Releases
+	// Stop watching. Will close the channel returned by ResultChan(). Releases
```
Comment (alexzielenski) [inline staging/src/k8s.io/apimachinery/pkg/watch/watch.go:30]: I think this wording was actually intended and correct. According to [godoc](https://tip.golang.org/doc/comment):

> As with packages (above) and funcs (below), doc comments for types start with complete sentences naming the declared symbol. An explicit subject often makes the wording clearer, and it makes the text easier to search, whether on a web page or a command line. For example:

See specifically the example for `Sort` showing a similar pattern:

```go
package sort

// Sort sorts data in ascending order as determined by the Less method.
// It makes one call to data.Len to determine n and O(n*log(n)) calls to
// data.Less and data.Swap. The sort is not guaranteed to be stable.
func Sort(data Interface) {
    ...
}
```

**Comment by alexzielenski** [comment]
Given my explanation above that this is likely the intended & correct godoc style, I'll be marking this closed. Please feel free to re-open and discuss further if you believe this is a mistake.

/close

**Comment by yanggangtony** [comment]
> Given my explanation above that this is likely the intended & correct godoc style, I'll be marking this closed. Please feel free to re-open and discuss further if you believe this is a mistake.
> 
> /close

@alexzielenski 
thanks for review..
I got it "godoc style" ..
Just close it , its fine to me..

**Pull request closed without merging**